test/test-creation.js

```diff
@@ -973,8 +973,8 @@ describe('JHipster generator', () => {
                 `${TEST_DIR}features/user/user.feature`
             ]);
             assert.noFile([
-                `${TEST_DIR}gatling/gatling.conf`,
-                `${TEST_DIR}gatling/logback.xml`
+                `${TEST_DIR}gatling/conf/gatling.conf`,
```

Comment:
Sorry, indeed I forgot to move the gatling.conf & the logback.xml inside this new folder.